components/CollectionCells/src/MDCCollectionViewCell.m

```diff
@@ -424,14 +424,19 @@ - (void)updateInterfaceForEditing {
 #pragma mark - Selecting
 
 - (void)setSelected:(BOOL)selected {
+  BOOL previousSelectedState = self.selected;
   [super setSelected:selected];
   if (selected) {
-    _editingSelectorImageView.image = [MDCIcons imageFor_ic_check_circle];
-    _editingSelectorImageView.tintColor = self.editingSelectorColor;
+    if (_editingSelectorImageView && previousSelectedState != selected) {
+      _editingSelectorImageView.image = [MDCIcons imageFor_ic_check_circle];
+      _editingSelectorImageView.tintColor = self.editingSelectorColor;
+    }
     self.accessibilityTraits |= UIAccessibilityTraitSelected;
   } else {
-    _editingSelectorImageView.image = [MDCIcons imageFor_ic_radio_button_unchecked];
-    _editingSelectorImageView.tintColor = HEXCOLOR(kCellGrayColor);
+    if (_editingSelectorImageView && previousSelectedState != selected) {
+      _editingSelectorImageView.image = [MDCIcons imageFor_ic_radio_button_unchecked];
+      _editingSelectorImageView.tintColor = HEXCOLOR(kCellGrayColor);
+    }
     self.accessibilityTraits &= ~UIAccessibilityTraitSelected;
   }
   _editingSelectorImageView.image =
```

Comment:
We should only call  the imageWithRenderingMode: if the image has changed.  I would expect the if blocks to look similar to the following:

```
if (_editingSelectorImageView && previousSelectedState != selected) {
     _editingSelectorImageView.image = [MDCIcons imageFor_ic_radio_button_unchecked];
     _editingSelectorImageView.tintColor = HEXCOLOR(kCellGrayColor);
     _editingSelectorImageView.image = 
         [_editingSelectorImageView.image imageWithRenderingMode:UIImageRenderingModeAlwaysTemplate]
 }
```